internal/jennies/java/templates/types/class.tmpl

```diff
@@ -0,0 +1,54 @@
+package {{ .Package }};
+
+{{ .Imports }}
+
+{{- range .Comments }}
+// {{ . }}
+{{- end }}
+public class {{ .Name }}{{ if .Extends }} extends {{ range $i, $e := .Extends }}{{ if gt $i 0 }}, {{ end }}{{ $e }}{{ end }}{{ end }}{{ if .Variant }} implements cog.variants.{{ .Variant }}{{ end }} {
```

Comment:
Variant sets the package here `cog.variants.` to avoid conflicts with classes with the same name.